<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QWidget" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_status">
     <property name="text">
      <string>영업중    보통   오전 9:00~ 오후 23:00</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
     </property>
     <property name="text">
      <string>영업상태 변경</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>교촌치킨 소촌점(123123)</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>즉각적인 매출성장을 경험하세요</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>66</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>IMG</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>노출과 매출 상승을 </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>만들어 보세요</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
      </property>
      <property name="text">
       <string>광고시작하기</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>고민중인 고객이 주문을</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>완료하게 만드세요</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
      </property>
      <property name="text">
       <string>맞춤형 쿠폰 발행하기</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QFrame" name="frame_3">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>오늘현황</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>매출액(진행중인 주문 포함)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>113,000원</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>5건</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>주문수(진행중인 주문 포함)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>22600원</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>평균 주문 금액</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>12000원 1건</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>거절/ 취소 주문</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>매일 오전6시에 초기화 됩니다.</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>202</width>
        <height>78</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_menu">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
         </property>
         <property name="text">
          <string>메뉴 관리</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_order">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
         </property>
         <property name="text">
          <string>주문 관리</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_time">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
         </property>
         <property name="text">
          <string>영업시간 관리</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_logout">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
         </property>
         <property name="text">
          <string>로그아웃</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frame_1">
     <property name="maximumSize">
      <size>
       <width>330</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0,0,0">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>170</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="LOGO.qrc">:/images/kyochon.jpg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_23">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>WOW 와우매장</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>쿠팡 부담 와우회원 혜택</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>무료배달</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>와우 + 즉시할인</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>즉시할인 매장 혜택 적용 중</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="LOGO.qrc"/>
 </resources>
 <connections/>
</ui>
